_includes/ee-linux-install-reuse.md

```diff
@@ -131,120 +127,84 @@ You can install Docker EE in different ways, depending on your needs:
-    instead of always using the latest. List the available versions.
-    This example uses the `sort -r` command to sort the results by version
-    number, highest to lowest, and is truncated.
+    a.  List and sort the versions available in your repo. This example sorts results by version number, highest to lowest, and is truncated:
 
     ```bash
     $ sudo yum list docker-ee  --showduplicates | sort -r
 
-    docker-ee.x86_64         {{ site.docker_ee_version }}.ee.2-1.el7.{{ linux-dist }}          docker-ee-stable-17.06
+    docker-ee.x86_64      {{ site.docker_ee_version }}.ee.2-1.el7.{{ linux-dist }}      docker-ee-stable-17.06
     ```
 
-    The contents of the list depend upon which repositories you have enabled,
-    and is specific to your version of {{ linux-dist-long }}
-    (indicated by the `.el7` suffix on the version, in this example). Choose a
-    specific version to install. The second column is the version string. You
-    can use the entire version string, but **you need to include at least to the
-    first hyphen**. The third column is the repository name, which indicates
-    which repository the package is from and by extension its stability level.
-    To install a specific version, append the version string to the package name
-    and separate them by a hyphen (`-`):
+    The list returned depends on which repositories you enabled, and is specific to your version of {{ linux-dist-long }} (indicated by `.el7` in this example).
 
-    > **Note**: The version string is the package name plus the version up to
-    > the first hyphen. In the example above, the fully qualified package name
-    > is `docker-ee-17.06.1.ee.2`.
+    b.  Install a specific version by its **fully qualified package name** which is the package name (`docker-ee`) plus the version string (2nd column) up to the hyphen, for example: `docker-ee-17.06.1.ee.2`
 
     ```bash
     $ sudo yum -y install <FULLY-QUALIFIED-PACKAGE-NAME>
     ```
 
-    Docker is installed but not started. The `docker` group is created, but no
-    users are added to the group.
+    Docker is installed but not started. The `docker` group is created, but no users are added to the group.
 
-{% if linux-dist == "centos" or linux-dist == "rhel" or linux-dist == "oraclelinux" %}
-3.  If you need to use `devicemapper`, follow the procedure in the
-    [devicemapper storage driver guide](/engine/userguide/storagedriver/device-mapper-driver.md#configure-direct-lvm-mode-for-production){: target="_blank" class="_" }
-    **before starting Docker**. For production systems using `devicemapper`,
-    you must use `direct-lvm` mode,
-    which requires you to prepare the block devices.
-{% endif %}
+3.  Start Docker:
 
-4.  Start Docker.
+    > If using `devicemapper`, ensure it is properly configured before starting Docker, per the [storage guide](/storage/storagedriver/device-mapper-driver/){: target="_blank" class="_" }.
```

Comment:
I've noticed a couple of notes like this. Make sure you use the new style

```
> Note title
>
> line 1
> line 2
```

Reply:
@joaofnfernandes , it takes up so much space. I thought this was one of 3 styles. ;)

Reply:
The first style should be considered legacy. It works because we have a lot of stuff written in the past using that style. Whenever we fix something or write a new piece we should be using the new style.

I also think it's a good thing that the new style takes a lot of space. Whenever you write a note it should feel ugly to you as a writer, so that we don't end up with docs with a ton of notes spread all over the place.

Reply:
I agree that too many notes is an eyesore and makes a page unreadable. I disagree that all notes should be as large and as ugly as possible. Within procedures/tasks, if I need a note, I do my best to keep it crisp and on one line. We can discuss offline.